### ISSUE TITLE
Images Missing and No Alt Text.

### ISSUE THREAD

**Issue by lucifertrj** [fullstackreact/30-days-of-react#190]
In the document file about the deployment the images are missing and without alt text.
https://github.com/fullstackreact/30-days-of-react/blob/master/day-28/doc/doc.mkd
